javascript/javascript1/week1/index.js

```diff
@@ -0,0 +1,124 @@
+/* Javascript week1 homework*/
+
+//Start of A future age calculator
+console.log("A future age calculator");
+const yearOfBirth = 1990;
+const yearFuture = 2040;
+const age = yearFuture - yearOfBirth;
+console.log(`You will be ${age} years old in ${yearFuture}`);
+//End of future age calculator
+
+//Start of A dog age calculator
+console.log("A dog age calculator");
+const dogYearOfBirth = 2000;
+const dogYearFuture = 2030;
+const dogYear = 50;
+let shouldShowResultInDogYears = true;
+if (shouldShowResultInDogYears) {
+	console.log(`Your dog will be ${dogYear} in ${dogYearFuture}`);
+} else {
+	console.log(
+		`Your dog will be ${dogYearFuture - dogYearOfBirth} in ${dogYearFuture}`
+	);
+}
+// End of A dog age calculator
+
+// Start of Housey pricey (A house price estimator)
+const peterHouse = {
+	width: 8,
+	depth: 10,
+	height: 10,
+	gardensize: 100,
+	house_cost: 2500000,
+};
+
+const juliaHouse = {
+	width: 5,
+	depth: 11,
+	height: 8,
+	gardensize: 70,
+	house_cost: 1000000,
+};
+
+function calculateVolumeInMeters(width, height, depth) {
+	volumeInMeters = width * height * depth;
```

Comment:
One very important thing: If you don't initialize your variables with `let` or `const` keywords, javascript will interpret it as a `global variable`, which can lead to unexpected results(can easily be overwritten) and make your code more difficult to debug. To avoid bad practice, try initializing them🙂

Reply:
yes sure i  will initialize variables . Thank you so much for your valuable feedback.

Reply:
No worries! Once you have done the changes, I can review it again and approve your pr😊

Reply:
i have done changes as per review. please check and let me know if there any updation in code. Thank you.